package/MDAnalysis/topology/PrimitivePDBParser.py

```diff
@@ -45,149 +45,33 @@
    :members:
    :inherited-members:
 
+..deprecated:: 0.15.0
+    PDBParser has been replaced with PrimitivePDBParser.
 """
+
 from __future__ import absolute_import, print_function
 
 import numpy as np
 import warnings
 
+from . import PDBParser
 from ..core.AtomGroup import Atom
 from .core import get_atom_mass, guess_atom_element
 from ..lib.util import openany
 from .base import TopologyReader
 
 
-class PrimitivePDBParser(TopologyReader):
-    """Parser that obtains a list of atoms from a standard PDB file.
-
-    See Also
-    --------
-    :class:`MDAnalysis.coordinates.PDB.PrimitivePDBReader`
 
-    .. versionadded:: 0.8
-    """
+class PrimitivePDBParser(PDBParser.PDBParser):
```

Comment:
Can't you rename this whole while to `PDBParser` and remove the old Bio Python PDB Topology parser?